### DISCUSSION TITLE
Custom Map Styles

### DISCUSSION THREAD

**Discussion by radh21301** [Freika/dawarich#198] [Ideas]
It would be awesome if we can use custom map styles. I prefer something other than the default ones we have now- something that is a little more pleasant to view. Different people have different tastes, and this would help cater to all of them.

**Option:**
To help give some context to the devs - Immich - a photo hosting app has a similar functionality. Hopefully this links helps somehow: https://immich.app/docs/guides/custom-map-styles/

We could allow setting a 'URL to a style.json map theme'  which would allow users to use whatever service that supports this style.

This is a style I like on [maptiler](https://www.maptiler.com/) (I definitely like this better!):
<img width="936" alt="image" src="https://github.com/user-attachments/assets/51dd7453-9b14-45f9-b718-526c3bef5902">

**Comment by Freika** [comment]
Just released 17 new styles that are available without api keys: https://github.com/Freika/dawarich/releases/tag/0.14.0

More will follow
